FirstProject/evaluation.html

```diff
@@ -0,0 +1,85 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <link rel="stylesheet" href="css/main.css">
+        <link rel="stylesheet" href="img/">
+        <!-- Latest compiled and minified CSS -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+
+        <!-- jQuery library -->
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+        <!-- Latest compiled JavaScript -->
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    </head>
+
+    <body>
+        <div>
+            <div class="row">
+                <nav class="col-md-12" style="border-bottom: 2px solid darkgrey; padding-top: 12px; padding-bottom: 14px;">
+                    <div class="col-md-4" style="text-align:center;">
+                        <img href="#" style="width: 40%;" src="img/logo-v2.png"/>
+                    </div>
+                    <div class="col-md-4">
+                        <a href="#">Evaluations</a>
+                        <a href="#" style="margin-left: 100px;">New Evaluation</a>
+                    </div>
+                    <div class="col-md-4">
+                        <a href="#" style="float:right;">Logout</a>
+                    </div>
+                </nav>
+            </div>
+        </div>
+<br/>
+<br/>
+<br/>
+    <div class="row">
+        <div class="container" style="margin-top: 120px;">
+            <table style="width:100%">
+                <tr>
+                    <th style="background-color: cornflowerblue; color: white; padding: 10px 0 10px 0; ">Nume</th>
+                    <th style="background-color: cornflowerblue; color: white;">Tehnologie</th>
+                    <th style="background-color: cornflowerblue; color: white;">Nivel</th>
+                    <th style="background-color: cornflowerblue; color: white;"></th>
+                </tr>
+                <tr>
+                    <td>Popescu Adrian</td>
+                    <td>Javascript</td>
+                    <td>Mid 2</td>
+                    <td>Detalii <span style="background-color: cornflowerblue; border-radius: 20px;">+</span></td>
```

Comment:
Any particular reason why you chose to use a `<span>` for the `+` button? Would you have other, possibly better, options?